deeplake/integrations/mmdet/mmdet_utils.py

```diff
@@ -85,20 +104,24 @@ def createDeeplakeIndex(self):
                 is_crowds = np.zeros_like(categories)
             img = {
                 "id": row_index,
-                "height": all_imgs[row_index][0],
-                "width": all_imgs[row_index][1],
+                "height": all_imgs[row_index].shape[0],
+                "width": all_imgs[row_index].shape[1],
```

Comment:
if all_imgs = self.img_shapes, then this .shape will make things incorrect right?